### PR TITLE
Added : wave footer

### PR DESCRIPTION
**Pull request by sameersrivastava13**
@smaranjitghose Please review. If you want me to change the color of the footer do let me know.

Screenshot (111)

Screenshot (112)

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -34,3 +34,111 @@ nav {
...
+}
+.fa {
+    display: inline-block;
+    font: normal normal normal 14px/1 FontAwesome;
+    font-size: inherit;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+    @import url("https://fonts.googleapis.com/css?family=Lato:400,400i,700");
```
Comment (smaranjitghose) [inline assets/css/style.css:49]: Please use Montserrat

File: index.html [hunk 2]
```diff
@@ -24,4 +24,51 @@
...
+           0 0 0 13 -9" result="goo" />
+      <xfeBlend in="SourceGraphic" in2="goo" />
+    </filter>
+     <path id="wave" d="M 0,10 C 30,10 30,15 60,15 90,15 90,10 120,10 150,10 150,15 180,15 210,15 210,10 240,10 v 28 h -240 z" />
+  </defs> 
+
+   <use id="wave3" class="wave" xlink:href="#wave" x="0" y="-2" ></use> 
+   <use id="wave2" class="wave" xlink:href="#wave" x="0" y="0" ></use>
+ 
+
```
Comment (smaranjitghose) [inline index.html:47]: Try to load it from a main.js file so that we can extend it to other pages

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -24,4 +24,51 @@
...
+  <circle class="drop drop2" cx="5.2" cy="5.1" r="7.5"  />
+  <circle class="drop drop3" cx="10.2" cy="5.3" r="9.2"  />
+    <circle class="drop drop4" cx="3.2" cy="5.4" r="8.8"  />
+  <circle class="drop drop5" cx="14.2" cy="5.1" r="7.5"  />
+  <circle class="drop drop6" cx="17.2" cy="4.8" r="9.2"  />
+  <use id="wave1" class="wave" xlink:href="#wave" x="0" y="1" />
+ </g>  
+</svg>
+
+  <div>Made with <a class="btn-floating pulse red"><i class="material-icons">favorite</i></a> in India for the students of the world &copy;<a href="">Doc2Pen, 2020 </a></div>
```
Comment (smaranjitghose) [inline index.html:72]: Just keep it "Made with 💖in India for the students of the world"  No copyright etc required

Comment (smaranjitghose) [inline index.html:72]: Avoid using Inline CSS as much as possible

File: assets/css/style.css [hunk 4]
```diff
@@ -34,3 +34,111 @@ nav {
...
+.drop {
+  fill: var(--col-deepblue);
+  xfill: #99000055;
+  animation: drop 3.2s linear infinite normal;
+  stroke: var(--col-deepblue);
+  stroke-width:0.5;
+  transform: translateY(25px) ;
+  transform-box: fill-box;
+  transform-origin: 50% 100%;
+}
```
Comment (smaranjitghose) [inline assets/css/style.css:90]: Put comments for each of them like for what section this written for

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (sameersrivastava13) [reply assets/css/style.css:49]: Done.

File: index.html (hunk 3 above)
Comment (sameersrivastava13) [reply index.html:72]: Done.

File: index.html (hunk 2 above)
Comment (sameersrivastava13) [reply index.html:47]: Done.

**Pull request merged**